plugins/modules/files/xattr.py

```diff
@@ -158,7 +158,7 @@ def _run_xattr(module, cmd, check_rc=True):
         if line.startswith('#') or line == '':
             pass
         elif '=' in line:
-            (key, val) = line.split('=')
+            (key, val) = line.split('=', maxsplit=1)
```

Comment:
```suggestion
            (key, val) = line.split('=', 1)
```
Python 2's split does not accept keyword arguments. This syntax works for both Python 2 and Python 3.